src/helpers/Constants.ts

```diff
@@ -20,12 +20,12 @@ isTablet = Platform.isPad || (getAspectRatio() < 1.6 && Math.max(screenWidth, sc
 
 function setStatusBarHeight() {
   const {StatusBarManager} = NativeModules;
-  statusBarHeight = 0; // so there will be a value for any case
-  statusBarHeight = isIOS ? 20 : StatusBarManager.HEIGHT;
-  if (isIOS) {
-    // override guesstimate height with the actual height from StatusBarManager
-    StatusBarManager.getHeight((data: any) => (statusBarHeight = data.height));
-  }
+  statusBarHeight = StatusBarManager.HEIGHT || 0; // So there will be a value for any case
```

Comment:
Why not apply the old logic instead of setting to 0 in case 'StatusBarManager.HEIGHT' returns null/undefined?

Reply:
Technically it works. We don't even need the 0 for that matter. 
Hmm The reason I didn't kept the old logic is cause I would prefer to remove it eventually if everything goes well.